core/src/test/scala/org/apache/spark/deploy/history/FsHistoryProviderSuite.scala

```diff
@@ -394,6 +393,30 @@ class FsHistoryProviderSuite extends SparkFunSuite with BeforeAndAfter with Matc
     }
   }
 
+  test("ignore hidden files") {
+    // FsHistoryProvider should ignore hidden files.  (It even writes out a hidden file itself
```

Comment:
A better test would be to write a valid log as a hidden file, and check that it wasn't read, if that's the goal of the change.